packages/server/src/DataSource.ts

```diff
@@ -89,3 +79,15 @@ export function getDataSource(): DataSource {
     }
     return appDataSource
 }
+
+const getDatabaseSSLFromEnv = () => {
+    if (process.env.DATABASE_SSL_KEY_BASE64) {
+        return {
```

Comment:
See comment above. getDatabaseSSLFromEnv should return
```
{
  ssl: {
    self signed ssl attributes
  }
}
```

and below should return 
{
  ssl: true
}

the final return of {} will return an empty object that will spread to nothing and allow pg node to continue to use PGSQLMODE as the lib is designed.

mysql2 does not appear to be impacted by using the same approach and should work by using the same spread methodology. mysql2 does not reference any env var internally.